postgres-appliance/ship_yesterday_pg_log_to_s3.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# the BusyBox version of `date` installed in Spilo lacks the ' --date "yesterday" ' option, so we process the date in code
```

Comment:
busybox is used only to decompress `/a.tar.xz`
After that you get a usual bash.